jit/v2/JIT_v2.0.0.yaml

```diff
@@ -2862,82 +2862,82 @@ components:
         - _40_units
```

Comment:
**Line 2846, 52, and 58**
 Forecasted container units of `20-foot` containers (40, and 45)

Can be replaced with something like the suggestion  below for line 2948?
"Forecasted units of 20-foot containers" 
"Forecasted units of 40-foot containers" 
"Forecasted units of 45-foot containers"